libfreerdp/common/assistance.c

```diff
@@ -563,9 +563,6 @@ BYTE* freerdp_assistance_encrypt_pass_stub(const char* password, const char* pas
 	if (!pbIn || !pbOut)
 		goto fail;
 
-	if ((EncryptedSize == 0) || (cbPassStubW > UINT32_MAX))
```

Comment:
Why remove this? The function is part of the API and might be used by external software, so all assumptions about arguments must hold

Reply:
I think that It's always false, here is my reasoning:
cbPassStubW changed only once here:
`cbPassStubW = (size_t)(status - 1) * 2UL;`
if the "status" will have its maximum, then:
`cbPassStubW == (size_t)(numeric_limits<int>::max() - 1) * 2UL == 4294967292`
and `4294967292 < UINT32_MAX (4294967295)`
so, expression cbPassStubW > UINT32_MAX is always false 
EncryptedSize is cbPassStubW + 4 and cbPassStubW always > 0 (because size_t), so (EncryptedSize == 0) is always false too :)

Reply:
ah yes, fell victim to diff folding :/